osxcollector/output_filters/summary_filters/html.py

```diff
@@ -327,4 +329,11 @@ def _print_val(self, val):
                 self._write('</li>')
             self._write('</ul>')
         elif isinstance(val, basestring) or isinstance(val, Number):
+            self._encode_val(val)
+
+    def _encode_val(self, val):
+        try:
             self._write(u'<span class="val">{0}</span>'.format(val))
+        except UnicodeDecodeError as err:
+            sys.stderr.write('Unicode decode error while processing the value: {0}\n{1}\n'.format(val, err))
+            self._write('<span class="val">{0}</span>'.format(val))
```

Comment:
The issue is that you can't mix unicode and raw bytes, so I think this might still fail. I think I would rather see
``val = unicode(val, errors='replace')`` which would replace the non-unicode characters with ? symbols.

```
>>> print unicode('\xe9\xdf\xf4 blah', errors='replace')
��� blah
```